### PR TITLE
Add refund script

### PR DESCRIPTION
**Pull request by TheSpaceCuber**
* Updated README for context (See the new readme [here](https://github.com/TheSpaceCuber/Themis/tree/add-refund-script))
* Script will be used in demo, but needs to be manually updated with addresses
* Basically 2 steps: Read data, invoke refund function with donor's data
* Remove refundAllCampaigns() because for now users handle their own refunds, hence that method doesn't work.

### CODE REVIEW

File: contracts/Campaign.sol [hunk 1]
```diff
@@ -115,25 +115,11 @@ contract Campaign {
...
-            //sender.transfer(value)
-            //emit hasRefunded(sender, value);
-        }
-        */
+    
+    function refund(address payable donor, uint256 amt) public distrustOnly {
+        require(amt > 0, "No amount to refund");
+        require(msg.sender == donor);
+        donor.transfer(amt);
+        emit hasRefunded(donor, amt);
```
Comment (alanlowzies) [inline contracts/Campaign.sol]: Just to clarify, this will only be called by the `refund.js` script?

**Review by alanlowzies — APPROVED** [review]
Looks alright. Just got one question about the `refund` function in `Campaign.sol`. Thanks!

File: contracts/Campaign.sol (hunk 1 above)
Comment (TheSpaceCuber) [reply contracts/Campaign.sol]: Yep, it's a "refund" POC after all

Comment (alanlowzies) [reply contracts/Campaign.sol]: Alright, got it. Thanks!

**Review by YitHien — APPROVED** [review]
LGTM!

**Pull request merged**